clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -1624,6 +1624,7 @@ Atom *PointerVariableConstraint::getAtom(unsigned AtomIdx, Constraints &CS) {
 // Brain Transplant params and returns in [FromCV], recursively.
 void FunctionVariableConstraint::brainTransplant(ConstraintVariable *FromCV,
                                                  ProgramInfo &I) {
+  llvm::errs() << "Brain Transplanting\n";
```

Comment:
This should not be unconditional; don't want to dump stuff to stderr without the verbose flag. (same below)